Taskfile.yml

```diff
@@ -407,6 +408,33 @@ tasks:
     cmds:
      - "{{.SCRIPTS_ROOT}}/generate-helm-manifest.sh {{.KUBE_RBAC_PROXY}} {{.LOCAL_REGISTRY_CONTROLLER_DOCKER_IMAGE}} {{.PUBLIC_REGISTRY}} {{.LATEST_VERSION_TAG}} `pwd`/{{.CONTROLLER_ROOT}}"
 
+  controller:install-helm:
+    desc: Generate and install helm chart on cluster
+    dir: "{{.CONTROLLER_ROOT}}"
+    cmds:
+    - "helm install --set azureSubscriptionID=$AZURE_SUBSCRIPTION_ID \
+      --set azureTenantID=$AZURE_TENANT_ID \
+      --set azureClientSecret=$AZURE_CLIENT_SECRET \
+      --set azureClientID=$AZURE_CLIENT_ID \
+      --set image.repository={{.LOCAL_REGISTRY_CONTROLLER_DOCKER_IMAGE}}
+      asov2 -n {{.ASO_NAMESPACE}} --create-namespace ./charts/azure-service-operator/"
+    - "kubectl rollout status -n {{.ASO_NAMESPACE}} deployment/azureserviceoperator-controller-manager"
+
+
+  controller:delete-helm:
+    desc: Delete helm release
+    cmds:
+      - "helm delete asov2 -n {{.ASO_NAMESPACE}}"
+
+  controller:test-helm-installation:
+    desc: Test helm chart installation and deletion
+    cmds:
+      - task: controller:install-cert-manager
+      - task: controller:build
+      - task: controller:docker-push-local
+      - task: controller:gen-helm-manifest
```

Comment:
After this you should put 
```
      # We need the below to wait until cert-manager is up, otherwise the subsequent installation of webhooks fails. See https://cert-manager.io/next-docs/installation/verify/
      - "cmctl check api --wait=2m"
```

See the `kind-create-with-podidentity` target